src/components/YourCompany/NameTranslations/NameTranslation.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+  <div id="name-translation">
+    <confirm-dialog
+      ref="confirmTranslationDialog"
+      attach="#name-translation"
+    />
+    <v-layout class="mt-3" v-if="!isEditing">
+      <v-flex md3>
+        <label><strong>Name Translation(s)</strong></label>
+        <v-flex md1>
+            <v-chip v-if="hasNameTranslationChange" x-small label color="#1669BB" text-color="white">
+                Corrected
+            </v-chip>
+          </v-flex>
+      </v-flex>
+      <v-flex md7 v-if="drafTranslations && drafTranslations.length">
+        <div v-for="(name, index) in drafTranslations" :key="`name_translation_${index}`">{{name}}</div>
+      </v-flex>
+      <v-flex md7 v-else>
+        No name translations
+      </v-flex>
+      <v-flex md2 class="align-right" v-if="!hasNameTranslationChange">
+        <v-btn
+          text color="primary"
+          @click="isEditing = true"
+        >
+          <v-icon small>mdi-pencil</v-icon>
+          <span>Correct</span>
+        </v-btn>
+      </v-flex>
+      <v-flex md2 class="align-right" v-else>
+        <v-btn
+          text color="primary" class="undo-name-translation"
+          @click="resetNameTranslations"
+        >
+          <v-icon small>mdi-undo</v-icon>
+          <span>Undo</span>
+        </v-btn>
+
+        <!-- More Actions Menu -->
+        <span class="more-actions">
+          <v-menu offset-y>
+            <template v-slot:activator="{ on }">
+              <v-btn
+                text small color="primary"
+                class="more-actions-btn"
+                v-on="on"
+              >
+                <v-icon>mdi-menu-down</v-icon>
+              </v-btn>
+            </template>
+            <v-list>
+              <v-list-item
+                class="actions-dropdown_item"
+                @click="isEditing = true"
+              >
+                <v-list-item-subtitle>
+                  <v-icon small>mdi-pencil</v-icon>
+                  <span class="ml-1">Correct</span>
+                </v-list-item-subtitle>
+              </v-list-item>
+            </v-list>
+          </v-menu>
+        </span>
+      </v-flex>
+    </v-layout>
+    <v-layout class="mt-3" v-else>
+      <v-flex md3>
+        <label><strong>Name Translation(s)</strong></label>
+      </v-flex>
+      <v-flex md9>
+        <v-layout>
+          <v-flex>
+            <p>Name translations must use the Latin Alphabet (English, French, etc.).
+              Names that use other writing systems must spell the name phonetically in English or French.
+            </p>
+            <v-btn outlined color="primary" @click="isAddingNameTranslation=true" :disabled="isAddingNameTranslation">
+              <v-icon>mdi-plus</v-icon>
+              <span>Add Name Translation</span>
+            </v-btn>
+          </v-flex>
+        </v-layout>
+        <v-layout pb-5>
```

Comment:
Does "pb-5" work here or does it need to be a class?

Also, I find it's better for lower elements to define a top margin (or padding) instead of for upper elements to define bottom whitespace. I find this makes layout easier since you never get unwanted whitespace below an element.